README.md

```diff
@@ -1,6 +1,7 @@
-# protondb-i18n
+# protondb-i18n-ko_KR
 
-A crowdsourced journey to translate [ProtonDB](https://www.protondb.com) ([News post](https://www.protondb.com/news/protondb-spricht-nun-auch-deutsch))
+[ProtonDB](https://www.protondb.com) 사이트에 대한 한국어 번역본입니다.
```

Comment:
we have a standard place for attribution in a section below this, please add yours according to convention, or i can add for you: https://github.com/bdefore/protondb-i18n#hall-of-fame